calcom/cal.com
# fix: Orgs/create child teams CAL-1986

## What does this PR do?


https://www.loom.com/share/f09ccd5533ab488cb32621a1a4fe590d

Fixes #9979



## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



- Create a new org
- Go to `settings/teams/new` and create a new team
    - In the DB, the new team should have the `parentId` of the org id.
- Go to `/teams`
    - You should see the create team prompts
    - In the DB change your role to "member"
    - The create team prompts should disappear
- Go to the settings page
    - As the admin you should see the create team button on the side
    - As a member that button should disappear
- Go to `settings/teams/new` as a member and try creating a team
    - You should receive an error
- Create a team as a normal user without an org 

## Mandatory Tasks

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
